docs/functional.adoc

```diff
@@ -1,6 +1,17 @@
 
 == `functional.hpp`
 
+https://github.com/intel/cpp-std-extensions/blob/main/include/stdx/functional.hpp[`functional.hpp`]
+contains
+https://en.cppreference.com/w/cpp/utility/functional/bind_front[bind_front] and
+https://en.cppreference.com/w/cpp/utility/functional/bind_front[bind_back].
```

Comment:
Is this the right link for bind_back?

Reply:
Perhaps surprising, but yes - they're on the same page.